Sources/SwiftProtobuf/Message+JSONAdditions.swift

```diff
@@ -34,6 +34,26 @@ public extension Message {
     return visitor.stringResult
   }
 
+  /// Returns a Data containing the UTF-8 JSON serialization of the message.
+  ///
+  /// Unlike binary encoding, presence of required fields is not enforced when
+  /// serializing to JSON.
+  ///
+  /// - Returns: A Data containing the JSON serialization of the message.
+  /// - Throws: `JSONEncodingError` if encoding fails.
+  func jsonUTF8Data() throws -> Data {
```

Comment:
I reimplemented `jsonString()` in terms of `jsonUTF8Data`.  This is not sufficient to remove `stringResult` since that's used in lots of other places (especially by the various implementations of `_CustomJSONCodable.encodedJSONString()`).

I also changed the performance harness to use `jsonUTF8Data` instead of `jsonString`.